smoke/esm/it.sh

```
@@ -1,16 +1,62 @@
+#  dockerCompose="docker compose"
+#fi
+
+#cat > .env <<EOF
+#PROJECT_BASE=$projectBase
+#EOF
+#mkdir -p $projectBase/.esm-cache/vd $projectBase/.esm-cache/esm
+#chmod -R oug+w $projectBase/.esm-cache/vd $projectBase/.esm-cache/esm
+#ls -la $projectBase/.esm-cache
+#$dockerCompose down || true
+#$dockerCompose up -d
+packageDir=${projectBase=}/dist/fireproof-core
```

Shellcheck (0.10.0)

[warning] packageDir appears unused. Verify use (or export if used externally).

(SC2034)